packages/cli/src/commands/mock.ts

```diff
@@ -1,43 +1,97 @@
-      signale.star('Dynamic example generation enabled.');
-    }
+    if (cluster.isMaster) {
+      cluster.setupMaster({ silent: true });
 
-    const server = createServer(spec, { mock: { dynamic } });
-    try {
-      const address = await server.listen(port, host);
+      const signaleInteractiveInstance = new signale.Signale({ interactive: true });
+      signaleInteractiveInstance.await({ prefix: chalk.bgWhiteBright.black('[CLI]'), message: 'Starting Prism…' });
 
-      if (server.prism.resources.length === 0) {
-        signaleInteractiveInstance.fatal('No operations found in the current file.');
-        this.exit(1);
+      if (dynamic) {
+        signale.star({
+          prefix: chalk.bgWhiteBright.black('[CLI]'),
+          message: `Dynamic example generation ${chalk.green('enabled')}.`,
+        });
       }
 
-      signaleInteractiveInstance.success(`Prism is listening on ${address}`);
+      if (multiprocess) {
+        signale.star({
+          prefix: chalk.bgWhiteBright.black('[CLI]'),
+          message: `Multi process ${chalk.green('enabled')}.`,
+        });
+
+        const worker = cluster.fork();
 
-      server.prism.resources.forEach(resource => {
-        signale.note(`${resource.method.toUpperCase().padEnd(10)} ${address}${resource.path}`);
-      });
-    } catch (e) {
-      signaleInteractiveInstance.fatal(e.message);
+        if (worker.process.stdout) {
+          pipeOutputToSignale(worker.process.stdout);
+        }
+      } else {
+        const logStream = new PassThrough();
+        const logInstance = createLogger('CLI', undefined, logStream);
+        pipeOutputToSignale(logStream);
+        await createPrismServerWithLogger(spec, dynamic, port, host, logInstance);
+      }
+    } else if (multiprocess && cluster.isWorker) {
+      const logInstance = createLogger('CLI');
+      await createPrismServerWithLogger(spec, dynamic, port, host, logInstance);
     }
   }
 }
+
+async function createPrismServerWithLogger(
+  spec: any,
+  dynamic: boolean,
+  port: number,
+  host: string | undefined,
+  logInstance: Logger,
+) {
+  const server = createServer(spec, { mock: { dynamic } }, logInstance.child({ name: 'HTTP SERVER' }));
+  try {
+    const address = await server.listen(port, host);
+    if (server.prism.resources.length === 0) {
+      logInstance.fatal('No operations found in the current file.');
+      cluster.worker.kill();
+    }
+    server.prism.resources.forEach(resource => {
+      logInstance.note(`${resource.method.toUpperCase().padEnd(10)} ${address}${resource.path}`);
+    });
+    logInstance.start(`Prism is listening on ${address}`);
+  } catch (e) {
+    logInstance.fatal(e.message);
+    cluster.worker.kill();
+  }
+}
+
+function pipeOutputToSignale(stream: Readable) {
+  stream.pipe(split(JSON.parse)).on('data', (logLine: LogDescriptor) => {
+    const logLevelType = logLevels.labels[logLine.level];
+
+    let prefix = LOG_COLOR_MAP[logLine.name].black(`[${logLine.name}]`);
```

Comment:
no need for `let`, let's have `const prefix = constructPrefix(...)` so that:
1. there is no mutation
2. `pipeOutputToSignale` is shorter

Reply:
Good point! Done in 8d9f6d2